test/test_Quaternion.jl

```diff
@@ -102,6 +102,23 @@ end
     @test convert(Octonion{Float64},Octonion(1,2,3,4,5,6,7,8)) === Octonion(1.0,2.0,3.0,4.0,5.0,6.0,7.0,8.0)
 end
 
+@testset "iszero" begin
+    @test iszero(Quaternion(0.0, 0.0, 0.0, 0.0))
+    @test !iszero(Quaternion(1.0, 0.0, 0.0, 0.0))
+    @test !iszero(Quaternion(0.0, 1.0, 0.0, 0.0))
+    @test !iszero(Quaternion(0.0, 0.0, 1.0, 0.0))
+    @test !iszero(Quaternion(0.0, 0.0, 0.0, 1.0))
+    @test !iszero(Quaternion(0.0, 0.0, 0.0, 0.0, true))
```

Comment:
We still ignoring the `norm` flag in `isone`. Should this fixed in this PR?
```julia
julia> isone(Quaternion(1,0,0,0,false))
true
```

Reply:
No, `iszero` was only added here because we needed it (when we ued it). I think a future PR would add `isone` and some of the other `isXXX` methods.